evaluator/evaluator_dependency_tracing_test.go

```diff
@@ -0,0 +1,288 @@
+package evaluator
+
+import (
+	"fmt"
+	"koko/lexer"
+	"koko/object"
+	"koko/parser"
+	"math/rand"
+	"sort"
+	"strconv"
+	"testing"
+)
+
+func testEvalAndGetDeps(input string, fname string) object.Object {
+	l := lexer.New(input)
+	p := parser.New(l)
+	program := p.ParseProgram()
+	env := object.NewEnvironment()
+	return Eval(program, env)
+}
+
+func strEquals(t *testing.T, testStr string, truthStr string) {
+	if testStr != truthStr {
+		t.Errorf("string has wrong value. got %s, want %s",
+			testStr, truthStr)
+	}
+}
+
+/*func assertProgramDidNotError(t *testing.T, res object.Object) object.Object {
```

Comment:
Remove?

Reply:
👍

Reply:
goodbye dead code. You will be missed and lived a full life from mid November until now